docs/configuration/config-file-reference.md

```diff
@@ -82,6 +82,14 @@ api:
   # CLI flag: -http.prometheus-http-prefix
   [prometheus_http_prefix: <string> | default = "/prometheus"]
 
+  # Enable logging of header specific context information
+  # CLI flag: -api.LogHeaders
+  [LogHeaders: <boolean> | default = false]
```

Comment:
The naming style of the config is off, it_should_be_snake_case :-) 

Also "LogHeaders" is very generic, we should have clearer name on what headers we are referring to.